lib/movements.js

```diff
@@ -239,22 +239,28 @@ class Movements {
     let cost = Math.SQRT2 // move cost
     const toBreak = []
 
-    const blockC = this.getBlock(node, dir.x, 0, dir.z)
+    const blockC = this.getBlock(node, dir.x, 0, dir.z) // Landing block or standing on block when jumping up by 1
     const y = blockC.physical ? 1 : 0
 
+    const block0 = this.getBlock(node, 0, -1, 0)
```

Comment:
You moved this out of a condition but don't use it elsewhere, why ?

Reply:
Its used for the relative height calculation at line 254 and 263

Reply:
ok